<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>959</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GudPyTreeView" name="objectTree">
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="objectStack"/>
    </item>
    <item>
     <widget class="QGroupBox" name="plottingGroupBox">
      <property name="minimumSize">
       <size>
        <width>431</width>
        <height>911</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="loadInputFile"/>
    <addaction name="loadConfiguration"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="runPurge"/>
    <addaction name="runGudrun"/>
    <addaction name="iterateGudrun"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="viewInputFile"/>
   </widget>
   <addaction name="menuLoad"/>
   <addaction name="menuRun"/>
   <addaction name="menuPlot"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadInputFile">
   <property name="text">
    <string>Load input file</string>
   </property>
  </action>
  <action name="loadConfiguration">
   <property name="text">
    <string>Load configuration</string>
   </property>
  </action>
  <action name="runPurge">
   <property name="text">
    <string>Run Purge</string>
   </property>
  </action>
  <action name="runGudrun">
   <property name="text">
    <string>Run Gudrun</string>
   </property>
  </action>
  <action name="viewInputFile">
   <property name="text">
    <string>View live input file</string>
   </property>
  </action>
  <action name="iterateGudrun">
   <property name="text">
    <string>Iterate Gudrun</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GudPyTreeView</class>
   <extends>QTreeView</extends>
   <header>src.gui.widgets.gudpy_tree</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
